overlord/snapstate/snapmgr.go

```diff
@@ -1009,3 +1030,62 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	t.SetStatus(state.UndoneStatus)
 	return nil
 }
+
+func (m *SnapManager) startSnapServices(t *state.Task) error {
+	st := t.State()
+	st.Lock()
+	defer st.Unlock()
+
+	ss, _, err := snapSetupAndState(t)
+	if err != nil {
+		return err
+	}
+	info, err := readInfo(ss.Name(), ss.SideInfo)
+	if err != nil {
+		return err
+	}
+
+	pb := &TaskProgressAdapter{task: t}
+	st.Unlock()
+	err = m.backend.StartSnapServices(info, pb)
+	st.Lock()
+	return err
+}
+
+func (m *SnapManager) stopSnapServices(t *state.Task) error {
+	st := t.State()
+	st.Lock()
+	defer st.Unlock()
+
+	ss, _, err := snapSetupAndState(t)
+	if err != nil {
+		return err
+	}
+	info, err := readInfo(ss.Name(), ss.SideInfo)
+	if err != nil {
+		return err
+	}
+
+	pb := &TaskProgressAdapter{task: t}
+	st.Unlock()
+	err = m.backend.StopSnapServices(info, pb)
+	st.Lock()
+
+	return err
+}
+
+func (m *SnapManager) doStartSnapServices(t *state.Task, _ *tomb.Tomb) error {
```

Comment:
If that's all they do (depends on previous point), we don't need these wrappers. Let's just use the real functions when registering the tasks.